platform/administer/templates/index.mdx

```diff
@@ -0,0 +1,55 @@
+
+Each template includes the base configuration for a virtual cluster—such as the Kubernetes version, control plane type, networking mode, and storage settings. Templates can also define resource constraints (CPU, memory, storage) to prevent overconsumption and ensure predictable usage.
+
+To support secure and consistent self-service, templates allow administrators to define which settings users can modify through parameters. For example, you can expose only a few fields—like the number of replicas or a set of approved storage classes—while keeping the rest of the configuration fixed. This helps reduce the risk of misconfiguration, but also enables flexibility.
+
+Templates can also include preinstalled applications, such as monitoring agents or ingress controllers, using Helm charts or Kubernetes manifests. Additional Kubernetes resources like `ConfigMaps`, `RBAC`, or `NetworkPolicies` can be applied to either the host cluster or the virtual cluster itself.
+
+Because templates can be versioned, they support upgrade workflows. When a template is updated, those changes can be propagated to any virtual clusters based on it. This allows teams to update cluster configurations and installed apps consistently across environments.
+
+
+## Template types
+
+vCluster Platform supports multiple types of templates. Each one serves a different purpose in the cluster lifecycle:
+
+| Template Type                | Custom Resource                                           | Purpose                                                                                                                   | Common Use Cases                                                                         |
+| ---------------------------- | --------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------- |
+| Virtual cluster              | `VirtualClusterTemplate`                                  | Specifies how virtual clusters are provisioned, including control plane settings, synchronization behavior, and metadata. | Provision virtual clusters with consistent architecture, features, and default settings. |
+| App                          | `AppTemplate`                                             | Defines reusable application configurations, such as Helm charts or container specifications.                             | Deploy standard applications across projects or spaces using consistent settings.        |
+| Namespace                    | `NamespaceTemplate`                                       | Predefines configurations for new namespaces, including role-based access control, resource quotas, and labels.           | Automatically apply baseline policies when a namespace is created.                       |
```

Comment:
The custom resource for namespaces is `SpaceTemplate` ([ref](https://github.com/loft-sh/loft-enterprise/blob/main/staging/src/github.com/loft-sh/api/v4/pkg/apis/management/v1/spacetemplate_types.go#L15))